armi/settings/settingsValidation.py

```diff
@@ -431,6 +431,33 @@ def _willBeCopiedFrom(fName):
             self.NO_ACTION,
         )
 
+        self.addQuery(
+            lambda: self.cs["shuffleSequenceFile"]
```

Comment:
If you _can_, it would be nice to use the global `CONF_SHUFFLE_SEQUENCE_FILE` variables for the settings in these validators. If you aren't using them because of import loops, that's fine.

Reply:
[ceec293](https://github.com/terrapower/armi/pull/2219/commits/ceec29390618d08d972964c41880182f276673cb)